lib/chef/knife/vsphere_datastore_list.rb

```diff
@@ -46,27 +29,47 @@ class Chef::Knife::VsphereDatastoreList < Chef::Knife::BaseVsphereCommand
          short: '-L',
          description: "Indicates whether to list VM's in datastore",
          boolean: true
+  option :pool,
+         long: '--pool pool',
+         short: '-h',
```

Comment:
Does this work? I thought `-h` is for help and is already implemented by `knife`.